### PR TITLE
[IMP] web_editor: Add rel="ugc" to some user generated links 

### PR DESCRIPTION
**Pull request by sebgeelen**
+ Odoo Editor Lib Build

See odoo task : https://www.odoo.com/web#action=333&active_id=1695&cids=1&id=2319575&menu_id=4720&model=project.task&view_type=form

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Zynton — APPROVED** [review]
Just don't forget to remove the console.log but otherwise looks good to me :+1:

File: addons/web_editor/static/src/js/wysiwyg/wysiwyg.js [hunk 1]
```diff
@@ -586,6 +587,10 @@ const Wysiwyg = Widget.extend({
                 if (!linkWidget.$link.length) {
                     linkWidget.$link = $(linkWidget.getOrCreateLink(this.$editable[0]));
                 }
+                if (this.options.userGeneratedContent) {
+                    data.rel = 'ugc';
+                }
+                console.log('save', data);
```
[the remark this answers is not in the document]
Comment (sebgeelen) [reply addons/web_editor/static/src/js/wysiwyg/wysiwyg.js:593]: :eyes:

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**